.github/workflows/json-validator.yaml

```diff
@@ -0,0 +1,30 @@
+name: Validate JSON Files
+
+on:
+  push:
+    paths:
+      - '**/*.json'
+  pull_request:
+    paths:
+      - '**/*.json'
+
+jobs:
+  validate-json:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Install jq
+        run: sudo apt-get install jq
+
+      - name: Validate JSON
+        run: |
+          for file in $(find . -type f -name '*.json' | grep -v -e '/venv/' -e '/.venv/'); do
+            jq . "$file" >/dev/null || { echo "Invalid JSON: $file"; exit 1; }
+            if [[ $(tail -c1 "$file" | wc -l) -eq 0 ]]
```

Comment:
this needs a `; then`

Reply:
I wonder how we could be testing this better. Thanks for noticing it though